src/test/java/net/smartcosmos/edge/things/resource/UpdateThingResourceTest.java

```diff
@@ -151,8 +153,6 @@ public void testThatUpdateNonexistentThingFails() throws Exception {
 
         willReturn(thingResponseEntity).given(restTemplate)
             .exchange(any(RequestEntity.class), eq(Void.class));
-        willReturn(metadataResponseEntity).given(restTemplate)
-            .exchange(any(RequestEntity.class), eq(RestMetadataCreateResponseDto.class));
```

Comment:
I have removed this mock and the related `ResponseEntity` since it is never used.